### DISCUSSION TITLE
Weekly mail report change percentages timeframe

### DISCUSSION THREAD

**Discussion by Tristan971** [plausible/analytics#1266] [Anything Else]
Hi, first and foremost thanks for Plausible, it's a really awesome piece of tech!

The weekly report email includes some change percentage numbers, which I assumed to be week-on-week, however I'm starting to think that this isn't the case?

See here the week of August 10th-16th
![1016](https://user-images.githubusercontent.com/875533/130426821-7499e6a7-e02c-4258-afd3-30f24d5a2019.png)

Followed by the week of August 17th-23rd
![1723](https://user-images.githubusercontent.com/875533/130426874-516afb53-b42c-404a-9dcd-27877f4efc3f.png)

The views actually went up week-on-week, yet it says that there was a decrease in %age.

From this I can only guess that the percentages are not week-on-week, but in such a case what are they?

Thanks

**Comment by ukutaht** [comment]
Thanks for reporting @Tristan971 !

Are you on self-hosted or using the Plausible Cloud platform?

**Reply by Tristan971** [reply]
This is on self-hosted (Plausible 1.3 and Clickhouse 21.3.2.5), sorry if it should have been in that category -- was thinking that it was more of a general question

That said, after searching a bit more it might simply be a side-effect of https://github.com/plausible/analytics/discussions/344 and the weekly email is compares an unfinished period of 7d against a past but complete period of 7d ? (but only for the percentages, not the absolute values... 🤔 )

---
As a footnote re: Plausible Cloud platform, alas we are non-profit so with our traffic amounts it's not an option :/

**Reply by ukutaht** [reply]
I remember fixing an off-by-one error with these percentage comparisons but I can't dig up the commit at this point. I _think_ this will get fixed in the next release. At least I cannot reproduce your issue on current master.

**Reply by Tristan971** [reply]
sounds good, I will patiently await the next release then 🙂
